### PR TITLE
Correction in docs/concepts/services-networking/network-policies/

### LINKED ISSUE #12430: Issue with k8s.io/docs/concepts/services-networking/network-policies/
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**

The current statement is as below:

> allows connections **from** any pod in the “default” namespace with the label “role=db” to CIDR 10.0.0.0/24 on TCP port 5978

**Proposed Solution:**
This should be changed to -
> allows connections **to** any pod in the “default” namespace with the label “role=db” to CIDR 10.0.0.0/24 on TCP port 5978

**Page to Update:**
https://kubernetes.io/...

### PR DESCRIPTION
**Pull request by daminisatya**
Fix #12430

### CODE REVIEW

**Comment by daminisatya** [comment]
\assign steveperry-53

File: content/en/docs/concepts/services-networking/network-policies.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ So, the example NetworkPolicy:
    * any pod in the "default" namespace with the label "role=frontend"
    * any pod in a namespace with the label "project=myproject"
    * IP addresses in the ranges 172.17.0.0–172.17.0.255 and 172.17.2.0–172.17.255.255 (ie, all of 172.17.0.0/16 except 172.17.1.0/24)
-3. allows connections from any pod in the "default" namespace with the label "role=db" to CIDR 10.0.0.0/24 on TCP port 5978
+3. allows connections to any pod in the "default" namespace with the label "role=db" to CIDR 10.0.0.0/24 on TCP port 5978
```
Comment (danwinship) [inline content/en/docs/concepts/services-networking/network-policies.md]: No, the text is correct as written; the example NetworkPolicy sets up both ingress and egress rules for the `role=db` pods, and the last item is explaining an egress rule.

Perhaps some additionally wording or emphasis could be added to clarify that.

Comment (daminisatya) [reply content/en/docs/concepts/services-networking/network-policies.md]: @danwinship Thank you for the review :) 

I will rephrase it to clear the confusion may be. Will update the PR soon 👍

Comment (daminisatya) [reply content/en/docs/concepts/services-networking/network-policies.md]: @danwinship I rephrased the sentence to clear the confusion. Can you review, please.

@cody-clark

File: content/en/docs/concepts/services-networking/network-policies.md [hunk 2]
```diff
@@ -92,11 +92,12 @@ __egress__: Each `NetworkPolicy` may include a list of whitelist `egress` rules.
 So, the example NetworkPolicy:
 
 1. isolates "role=db" pods in the "default" namespace for both ingress and egress traffic (if they weren't already isolated)
-2. allows connections to TCP port 6379 of "role=db" pods in the "default" namespace from:
+2. (ingres rules) allows connections to all pods in the “default” namespace with the label “role=db” on TCP port 6379 from:
```
Comment (danwinship) [inline content/en/docs/concepts/services-networking/network-policies.md]: "ingress" and "egress" both end with a double "s". (But other than that, the edits are factually correct now.)
